index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Portfolio</title>
```

Comment:
- [ ] Kindly consider using a descriptive page title as it will help for `SEO`. In that way your page will be easily accessible on the internet.
`Hint: You may use hiwima's portfolio for example which makes more sense that portfolio only` :smile: